mutis/tests/test_correlation.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+import numpy as np
+from mutis.signal import Signal
+from mutis.correlation import Correlation
+
+
+@pytest.fixture
+def corr():
+    times1 = np.linspace(2, 6, 40)
+    values1 = np.sin(times1)
+    times2 = np.linspace(8, 12, 40)
+    values2 = np.sin(times2)
+    signal1 = Signal(times1, values1, "lc_gen_psd_nft")
+    signal2 = Signal(times2, values2, "lc_gen_psd_nft")
+    return {
+        "fail": Correlation(signal1, signal2, "test"),
+        "welsh_ab": Correlation(signal1, signal2, "welsh_ab"),
+        "kroedel_ab": Correlation(signal1, signal2, "kroedel_ab"),
+        "numpy": Correlation(signal1, signal2, "numpy"),
+        "fwelsh": Correlation(signal1, signal2, "fwelsh"),
+        "fkroedel": Correlation(signal1, signal2, "fkroedel"),
+    }
+
+
+def test_gen_synth(corr):
```

Comment:
What are all these tests exactly testing? there should be some asserts

Reply:
They are not numerical assessments, but functional tests (i.e. code does not break)

Reply:
I have added numerical assessment tests there where it has some meaning. In plot functions we just do functional tests, and where random values are returned we check for the right shape or length of the array.